src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -2,18 +2,90 @@
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int INITIAL_DEFAULT_CAPACITY = 16;
+    private static final double DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int MULTIPLICATION_FACTOR = 2;
+    private int size;
+    private Node<K, V>[] table;
+
+    MyHashMap() {
+        table = new Node[INITIAL_DEFAULT_CAPACITY];
+    }
+
     @Override
     public void put(K key, V value) {
+        if (size > table.length * DEFAULT_LOAD_FACTOR) {
+            resize();
+        }
+        int bucket = getHash(key);
+        Node<K, V> currentNode = table[bucket];
+        Node<K, V> newNode = new Node<>(key, value, null);
```

Comment:
You are not using the diamond operator here. It should be `new Node<>(key, value, null);` to take advantage of Java's type inference.